worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/FastSchematicReader.java

```diff
@@ -109,10 +109,10 @@ private CompoundTag fixBlockEntity(CompoundTag tag) {
         if (fixer == null || dataVersion == -1) {
             return tag;
         }
-        //FAWE start - BinaryTag
-        return (CompoundTag) AdventureNBTConverter.fromAdventure(fixer.fixUp(
+        //FAWE start - LinTag
```

Comment:
Why did this class had fawe annotations in the first place if it's not an upstream class?

Reply:
You wanna remove from the FastReader / (Writer?) as well?

Reply:
different PR but I take it lol

Reply:
Ah I checked out the wrong branch. Oh well